subprojects/docs/src/docs/userguide/dep-man/02-declaring-dependency-versions/dependency_locking.adoc

```diff
@@ -201,6 +208,11 @@ The goal is to have only a single lock file per project, which contains the lock
 By default, the file is named `gradle.lockfile` and is located inside the project directory.
 The lock state for the buildscript itself is found in a file named `buildscript-gradle.lockfile` inside the project directory.
 
+[NOTE]
+====
+Because feature previews are applied after Gradle settings are configured, it is impossible to leverage a single lockfile for the settings configurations locking.
```

Comment:
I don't quite understand this comment. And I don't think the reason (startup order) is relevant to user.

Reply:
So this is in the section "Single lock file per project"

It describes a feature preview that changes the lock file format to be 1 file per project instead of 1 per configuration.
However, if you attempt to use that feature, settings level locking will still use the regular format because the feature preview is discovered after setting up the `Settings` object.

I will rework the sentence so that the reason shows up last and not first. But I believe it worthwhile to explain the why.

Make sense?